setuptools/tests/test_distutils_adoption.py

```diff
@@ -67,3 +67,12 @@ def test_distutils_local(venv):
     preferred.
     """
     assert venv.name in find_distutils(venv, env=dict()).split(os.sep)
+
+
+@pytest.mark.xfail('IS_PYPY', reason='pypy imports distutils on startup')
+def test_distutils_invalid_option(venv):
+    """
+    Ensure local distutils is used even when an invalid option is provided.
+    """
+    env = dict(SETUPTOOLS_USE_DISTUTILS='xxx')
```

Comment:
The problem here is that the test above (and also in `_distutils_hack`) doesn't check against all valid values. Probably, there should be a check that the allowed values are "stdlib" and "local" and the behavior with any other value is undefined.

Reply:
Undefined means to throw an exception if any other values is provided?

Reply:
Right. I was unclear.

What I mean to say is when the value is anything but "stdlib" or "local", there's no guaranteed behavior, so the test should assert that `find_distutils` could contain venv.name or not. I don't want "xxx" to be a synonym for "local" as that encumbers all those possible values. If there was ever a need for "xxx" to mean a third option, that would not be possible without changing the meaning of "xxx", which would be a backward-incompatible change ("xxx" would mean "local" before the change and something else after).

I guess what I'm realizing now is that I don't want this change.